README.md

```diff
@@ -17,8 +19,8 @@ Type the following commands into SSH, line by line.
 
 ```bash
 cd /var/www/html/admin/style/vendor/
-sudo git clone https://github.com/thomasbnt/Night_Pihole.git
-cd Night_Pihole
+sudo git clone https://github.com/thomasbnt/Night_Pi-hole.git
+cd Night_Pi-hole
```

Comment:
Why do you rename this folder?

Reply:
I’ve deleted the Speedtest parts. That and the renaming was me being too keen and ‘excited’ for this. (Please forgive me, I’m pretty new to this GitHub thing.)

Reply:
No problem and don't worry! Welcome on GitHub and thanks for contribute at this project!